### DISCUSSION TITLE
The margins of the document are too large

### DISCUSSION THREAD

**Discussion by NelloHo** [vkbo/novelWriter#1373] [Help] [answered]
As the title says, my novelwriter looks like this :

![image](https://user-images.githubusercontent.com/99171663/220293977-2015d459-1842-4cfe-9472-ad5bda93a63b.png)

but I want the margin be smaller so that it can display more words in one line , just like the demo from novelwriter website:

![image](https://user-images.githubusercontent.com/99171663/220294303-100380aa-e933-435d-aa1c-e56a0ae74ffe.png)

Also , I'd like to change the font to bigger size (Tools -> preference -> font size only change the font size of the GUI). 
 
Pls help me ~ 
Thanks

**Comment by vkbo** [comment, marked as answer]
Please look through the different settings in the Preferences. There are multiple pages of settings. The font size for documents (editor and viewer) is set in the Documents part of Preferences. You will also find the text margin settings there.

![image](https://user-images.githubusercontent.com/1619840/220299423-a97b2477-2d80-482b-b430-6ccaca4dcdb5.png)

**Comment by NelloHo** [comment]
Thank you !

**Reply by vkbo** [reply]
Please don't write your response as a new answer. Use the reply box.
